migration-tool/cmd/prom-migrator/main.go

```diff
@@ -181,6 +182,34 @@ loop:
 	log.Info("msg", "exiting!")
 }
 
+type headersFlags struct {
```

Comment:
Nit: Consider moving this outside of main since it deals with very specific things